src/v/cluster/scheduling/constraints.h

```diff
@@ -10,9 +10,13 @@
  */
 
 #pragma once
+#include "cluster/health_monitor_types.h"
```

Comment:
health_monitor_types.h can be dropped I think

Reply:
fixed